### PR TITLE
Proper PR: Communication changes

### PR DESCRIPTION
**Pull request by devdo-eu**

### CODE REVIEW

File: TCP_Server.py [hunk 1]
```diff
@@ -5,6 +5,34 @@
 logging.basicConfig(level=logging.INFO, format='[%(asctime)s] %(message)s')
 
 
+class Connection:
+    def __init__(self, conn):
+        self.EOT = '\n\x04\n'
+        self.conn = conn
```
Comment (ppolakiewicz) [inline TCP_Server.py:11]: I know it is late and I cling, but shouldn't these fields be private?

Comment (devdo-eu) [reply TCP_Server.py:11]: more sources/code needs refactor - so.... i suggest another PR for private fields and unification of source notation.

Comment (ppolakiewicz) [reply TCP_Server.py:11]: Ok, I trust you :)

**Review by ppolakiewicz — APPROVED** [review]

**Comment by devdo-eu** [comment]
Thx ;)

**Pull request merged**